addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/discovery/NetworkDiscoveryService.java

```diff
@@ -23,25 +28,47 @@
 import org.eclipse.smarthome.core.thing.ThingUID;
 import org.openhab.binding.network.service.DiscoveryCallback;
 import org.openhab.binding.network.service.NetworkUtils;
+import org.osgi.service.component.ComponentContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * The {@link NetworkDiscoveryService} is responsible for discovering devices on
  * the current Network. It uses every Network Interface which is connected to a network.
+ * It tries common TCP ports to connect to, ICMP pings and ARP pings.
  *
+ * @author David Graeff

```

Comment:
Most of time I have seen it other way round. The last person that changes a file comes first (makes actually sense if you want to contact a person for asking for implementation details).